lib/Remotery.c

```diff
@@ -9284,8 +9345,13 @@ RMT_API void _rmt_PropertyFrameResetAll()
     {
         return;
     }
-        
+
     mtxLock(&g_Remotery->propertyMutex);
+    if (g_Settings.property_handler != NULL)
```

Comment:
I think `_rmtPropertySnapshotAll` is a better place for this as that's when the data gets sent to the viewer and logged. Potentially that means that the callback should be referenced as a "snapshot callback" as well.

Reply:
I agree. It was a case of me reading poorly and not seeing that the function existed 🙄